api/db/changelog/db.changelog-246-add-vwb-egress-event-columns.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet author="yonghao" id="db.changelog-246-add-is-vwb-egress-event-column">
+        <addColumn tableName="egress_event">
+            <column name="is_vwb" type="boolean" />
```

Comment:
do we need a separate field is_vwb or we can use the fact that if vwb_workspace_id is not empty then it's a vwb event?

Reply:
good point!  done